### PR TITLE
l10n: Remove spaces

### PR DESCRIPTION
**Pull request by Valdnet**
Signed-off-by: Valdnet <47037905+Valdnet@users.noreply.github.com>

### CODE REVIEW

**Comment by szaimen** [comment]
Thanks Valdnet but it also works with spaces. (See the string replaces below)

**Comment by Valdnet** [comment]
@szaimen Of course it works, but for the translator these are extra spaces in the translation (this is a spelling mistake). Additionally, it is not consistent with the rest of the code.

Sample text strings with a link without spaces.
https://github.com/nextcloud/server/blob/82f602e8575c213c5a49bee2dc89bbf18bce86f1/core/js/setupchecks.js#L155
https://github.com/nextcloud/server/blob/82f602e8575c213c5a49bee2dc89bbf18bce86f1/core/js/setupchecks.js#L186

**Comment by szaimen** [comment]
Fine by me then

**Review by szaimen — APPROVED** [review]

File: core/js/setupchecks.js [hunk 1]
```diff
@@ -197,7 +197,7 @@
 					}
 					if (!data.wasEmailTestSuccessful) {
 						messages.push({
-							msg: t('core', 'You have not set or verified your email server configuration, yet. Please head over to the {mailSettingsStart} Basic settings {mailSettingsEnd} in order to set them. Afterwards, use the "Send email" button below the form to verify your settings.',)
+							msg: t('core', 'You have not set or verified your email server configuration, yet. Please head over to the {mailSettingsStart}Basic settings{mailSettingsEnd} in order to set them. Afterwards, use the "Send email" button below the form to verify your settings.',)
 							.replace('{mailSettingsStart} ', '<a href="' + OC.generateUrl('/settings/admin') + '">')
 							.replace(' {mailSettingsEnd}', '</a>'),
```
Comment (szaimen) [inline core/js/setupchecks.js:202]:
```suggestion
							.replace('{mailSettingsStart}', '<a href="' + OC.generateUrl('/settings/admin') + '">')
							.replace('{mailSettingsEnd}', '</a>'),
```

**Review by CarlSchwan — APPROVED** [review]

**Pull request merged**